src/main/java/com/tileman/TilemanModePlugin.java

```diff
@@ -213,15 +225,70 @@ protected void shutDown() {
         points.clear();
     }
 
-    private Collection<GroundMarkerPoint> getPoints(int regionId) {
-        return getGroundMarkerConfiguration(REGION_PREFIX + regionId);
+    public void importGroundMarkerTiles() {
+        // Get and store all the Ground Markers Regions
+        List<String> groundMarkerRegions = removeRegionPrefixes(configManager.getConfigurationKeys("groundMarker.region"));
+        // If none, Exit function
+
+        // Get and store array list of existing Tileman World Regions (like updateTileCounter does)
+        List<String> tilemanModeRegions = removeRegionPrefixes(configManager.getConfigurationKeys(CONFIG_GROUP + ".region"));
+
+        // CONVERSION
+        // Loop through Ground Marker Regions
+        for (String region: groundMarkerRegions) {
+            // Get Ground Markers Region's Tiles
+            ArrayList<TilemanModeTile> groundMarkerTiles =
+                    new ArrayList<>(getConfiguration("groundMarker", REGION_PREFIX + region));
+            // If region already exists in Tileman World Regions Array:
+            if(tilemanModeRegions.contains(region)) {
+                // Create Empty ArrayList for Region;
+                // Get Tileman Region's tiles and add them to the region array list
+                ArrayList<TilemanModeTile> regionTiles = new ArrayList<>(getTiles(Integer.parseInt(region)));
+
+                // Create int for regionOriginalSize;
+                // Set regionOriginalSize to arraylists length
+                int regionOriginalSize = regionTiles.size();
+                int nonUniqueTiles = 0;
```

Comment:
Can delete this. Doesn't look like we use it anywhere

Reply:
Removing, was just using for debugging